packages/sequelize-data-provider/src/sequelizeDataProvider.js

```diff
@@ -66,26 +62,11 @@ async function getRelatedEntityIds(entityName, originalEntityId, relationEntityN
 }
 
 async function getRelatedEntities(entityName, originalEntityId, relationFieldName, args) {
-  const fieldType = getFieldType(schema, entityName, relationFieldName);
-  const argsAfterLimitFiltering = omitLimitArgsIfRequired(entityName, relationFieldName, args);
-  const sqFilter = args && openCrudToSequelize(argsAfterLimitFiltering, upperFirst(fieldType));
-  let include = {
-    model: model(fieldType),
-    as: relationFieldName,
-    required: true
-  };
-  if (sqFilter) {
-    include = {
-      ...include,
-      ...sqFilter
-    };
-  }
-  const originalEntity = await model(entityName).findOne({
-    where: { id: originalEntityId },
-    include: omit(include, ['attributes'])
-  });
-  let relatedEntities = originalEntity ? originalEntity[relationFieldName] : [];
-  relatedEntities = limitAfterFetch(args, relatedEntities);
+  const relatedEntityName = getFieldType(schema, entityName, relationFieldName);
```

Comment:
this is the heart of the PR ... and the idea was originated by a Sigma developer (aka Benny's friends) ... I just did the dirty work of cleaning up and adding some tests (;